azkaban-db/src/main/sql/create.project_versions.sql

```diff
@@ -1,13 +1,14 @@
 CREATE TABLE project_versions (
-  project_id  INT         NOT NULL,
-  version     INT         NOT NULL,
-  upload_time BIGINT      NOT NULL,
-  uploader    VARCHAR(64) NOT NULL,
-  file_type   VARCHAR(16),
-  file_name   VARCHAR(128),
-  md5         BINARY(16),
-  num_chunks  INT,
-  resource_id VARCHAR(512) DEFAULT NULL,
+  project_id           INT           NOT NULL,
+  version              INT           NOT NULL,
+  upload_time          BIGINT        NOT NULL,
+  uploader             VARCHAR(64)   NOT NULL,
+  file_type            VARCHAR(16),
+  file_name            VARCHAR(128),
+  md5                  BINARY(16),
+  num_chunks           INT,
+  resource_id          VARCHAR(512)  DEFAULT NULL,
+  startup_dependencies MEDIUMBLOB    DEFAULT NULL,
```

Comment:
Can you provide AJAX API to fetch the cooked dependencies?

Reply:
Discussed offline - dropping. You can get the startup dependencies for a project version by downloading the project zip.